src/components/node-list/index.js

```diff
@@ -219,9 +192,6 @@ export const mapDispatchToProps = (dispatch) => ({
   onToggleModularPipelineActive: (modularPipelineIDs, active) => {
     dispatch(toggleModularPipelineActive(modularPipelineIDs, active));
   },
-  onToggleModularPipelineFilter: (modularPipelineIDs, enabled) => {
-    dispatch(toggleModularPipelineFilter(modularPipelineIDs, enabled));
-  },
```

Comment:
This action would be needed for focus mode implementation, please retain this - you can add a comment to reference this if needed